recipes/python_packaging/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.47.0"
+
+
+class PackageConan(ConanFile):
+    name = "python_packaging"
```

Comment:
We should have consistence naming. #23098 used a hypen instead of underscore.
Personally, I would prefer a hypen. It is also closer to how we handle double package names.

Reply:
I used a hyphen to stay as close as possible to how Linux distributions manage Python packages, fyi.